dev.sh

```diff
@@ -341,6 +347,6 @@ action=${1-}
 if [[ -n $@ ]];then shift;fi
 
 case $action in
-    build|up|down|ps|checks|test_all|unittest|test|composer_update|notice) do_$action "$@";;
+    build|up|down|ps|checks|test_all|unittest|test|composer_update|notice|clean) do_$action "$@";;
```

Comment:
May be you can add a little description above (like line 334 just after `unittest ...`